tests/integration/vlan_test.py

```diff
@@ -35,6 +37,47 @@ def test_add_and_remove_vlan(eth1_up):
     assert not current_state[INTERFACES]
 
 
+def test_add_down_remove_vlan(eth1_up):
+    up = yaml.load("""
```

Comment:
Can you please move these `""...""` to be constants? (not the conversion to a dict)

Reply:
changed